docs/question/template.md

```diff
@@ -112,7 +112,7 @@ You can use this syntax to render over an array of strings (the `.` represents t
 
 !!! info "HTML rendering"
 
-    You can use triple-braces (e.g. `{{{params.html}}}`) to substitute raw HTML. This should only be used for content that is defined by the instructor. If you want to render student-provided HTML, you can use the [`<pl-xss-safe>` element](../elements.md#pl-xss-safe-element).
+    By default, Mustache escapes special characters when using double braces (`{{ }}`). You can use triple-braces (e.g. `{{{params.html}}}`) to substitute raw HTML (and perform no escaping). This should only be used for content that is defined by the instructor. If you want to render student-provided HTML, you can use the [`<pl-xss-safe>` element](../elements.md#pl-xss-safe-element).
```

Comment:
I think you should use the same `params.html` example in both parenthesized sections, just with different brace levels.